ui/open-api-ui/src/main/resources/template/index.html

```diff
@@ -30,6 +30,7 @@
                             ${urlSection},
                             dom_id: '${domId}',
                             deepLinking: ${deepLinking},
+                            queryConfigEnabled: ${queryConfigEnabled},
```

Comment:
It seems that this will output an invalid JSON? Maybe this needs to be wrapped in an `{#if queryConfigEnabled}`?

Reply:
THis is not using qute. Basic String replace. There is no template engine here :)

Reply:
If the value is null, the whole line should be omitted.

Reply:
> If the value is null, the whole line should be omitted.

Hm, if a basic String replace is performed, this needs to be changed then @mfpc

Reply:
I think this isn't handled yet, otherwise it will generate an invalid JSON afaik

Reply:
No it's ok. The String replace remove lines with null.